From a29c108f0df57c48ab97f7ba81e82c26d7e72113 Mon Sep 17 00:00:00 2001
From: FrancescoFerraro <francesco.f@variscite.com>
Date: Mon, 5 Dec 2022 07:36:11 +0100
Subject: [PATCH 2/2] plat: imx8mp: Add D1W permission to fix Cortex-M7 start
 failure

Without this patch the M7 doesen't start causing kernel boot failure
The issue is introduced by commit 32f8832

Signed-off-by: FrancescoFerraro <francesco.f@variscite.com>
---
 plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
index 0ecd6fc13..714f19cde 100644
--- a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
@@ -63,7 +63,7 @@ static const struct imx_rdc_cfg rdc[] = {
 
 	/* peripherals domain permission */
 	RDC_PDAPn(RDC_PDAP_WDOG1, D0R | D0W),
-	RDC_PDAPn(RDC_PDAP_RDC, D0R | D0W | D1R),
+	RDC_PDAPn(RDC_PDAP_RDC, D0R | D0W | D1R | D1W),
 
 	/* memory region */
 	RDC_MEM_REGIONn(40, 0x00000000, 0x50000000, ENA|D3R|D3W|D2R|/*D2W|*/D1R|D1W|D0R|D0W),
-- 
2.25.1

